permission-management/clone-object-permissions-wizard.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Clone Object Permissions Wizard
+description: This article describes how to use the Clone Object Permissions Wizard in order to equalize principal and group permissions on two different SharePoint objects.   
```

Comment:
equalize mi je dosta čudna riječ u ovom kontekstu. iako je možda točna

Reply:
to match možda?